axum-extra/CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning].
 
 # Unreleased
 
-- None.
+- **fixed:** `Option` and `Result` are now supported in typed paths ([#1001])
```

Comment:
This sounds like it's about fields inside the type that derives `TypedPath`. Maybe sth. like this? 

```suggestion
- **fixed:** `Option` and `Result` are now supported in typed path route handler parameters ([#1001])
```